<ui version="4.0" >
 <class>PhoneWindow</class>
 <widget class="QMainWindow" name="PhoneWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <widget class="QTableView" name="callList" >
    <property name="geometry" >
     <rect>
      <x>70</x>
      <y>30</y>
      <width>491</width>
      <height>271</height>
     </rect>
    </property>
    <property name="alternatingRowColors" >
     <bool>true</bool>
    </property>
    <property name="selectionBehavior" >
     <enum>QAbstractItemView::SelectItems</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="addCall" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>400</x>
      <y>370</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text" >
     <string>Add New</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="number" >
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>370</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="callMade" >
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>420</y>
      <width>201</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text" >
     <string>Call made at current time</string>
    </property>
    <property name="checked" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeEdit" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>260</x>
      <y>450</y>
      <width>118</width>
      <height>28</height>
     </rect>
    </property>
    <property name="calendarPopup" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>450</y>
      <width>110</width>
      <height>28</height>
     </rect>
    </property>
    <property name="maximumDateTime" >
     <datetime>
      <hour>23</hour>
      <minute>59</minute>
      <second>59</second>
      <year>7999</year>
      <month>11</month>
      <day>30</day>
     </datetime>
    </property>
    <property name="displayFormat" >
     <string>d MMM yyyy</string>
    </property>
    <property name="calendarPopup" >
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar" />
 </widget>
 <resources/>
 <connections/>
</ui>
